packages/fiber/src/core/hooks.native.ts

```diff
@@ -4,8 +4,18 @@ import { Asset } from 'expo-asset'
 import { readAsStringAsync } from 'expo-file-system'
 import { decode } from 'base64-arraybuffer'
 import { suspend, preload, clear } from 'suspend-react'
-import { buildGraph, ObjectMap, is } from '../core/utils'
-import { Extensions, LoaderResult, BranchingReturn, useStore, useThree, useFrame, useGraph } from '../core/hooks'
+import { buildGraph, ObjectMap, is } from './utils'
+import {
+  Extensions,
+  LoaderResult,
+  BranchingReturn,
+  useStore,
+  useThree,
+  useFrame,
+  useGraph,
+  // explicit file extension so it doesn't loop
+  // @ts-ignore
+} from './hooks.ts'
```

Comment:
This import may not transpile well, but we need to transclude (and not bundle) these files anyway.